common/scylla_configure.py

```diff
@@ -161,11 +162,16 @@ def start_scylla_on_first_boot(self):
             LOGGER.info("Disabling Scylla start on first boot")
             self.DISABLE_START_FILE_PATH.touch()
 
-    def create_devices(self):
+    def create_devices(self, dry_run=False):
```

Comment:
As far as I understand `dry_run` arg is created only for testing.
In such case you should use `unittest.mock.patch` in the tests. See example below